.eslintrc.js

```diff
@@ -187,7 +191,19 @@ module.exports = {
     'react/no-unstable-nested-components': ['error', { allowAsProps: true }],
   },
   overrides: [
-    ...baseline.overrides,
+    ...baseline.overrides.filter(
+      (override) => !override.extends?.includes('plugin:mocha/recommended'),
+    ),
+    {
+      ...mochaPluginOverride,
+      extends: [],
+      rules: Object.entries(mochaPluginOverride.rules).reduce((acc, [key, value]) => {
+        if (!key.includes('mocha')) {
```

Comment:
Explicitly altering the test rules to avoid the need for eslint-plugin-mocha`, while monorepo still "mandates" it. 🙈